src/main/java/annotation/WebbTestsRunner.java

```diff
@@ -0,0 +1,88 @@
+package annotation;
+
+import java.lang.reflect.Method;
+
+public class WebbTestsRunner {
+
+    public static void main(String[] args) {
+        startTests();
+    }
+
+    private static void startTests() {
+        Class testsClass = WebbTests.class;
+        WebbTests mainObj = new WebbTests();
+        Method[] methods = testsClass.getDeclaredMethods();
+
+        printBeforeClass(methods, mainObj);
+
+        for (Method method : methods) {
+            String methodName = method.getName();
+            if (method.isAnnotationPresent(WebbTest.class)) {
+                WebbTest anno = method.getDeclaredAnnotation(WebbTest.class);
+                printBeforeMethod(methods, mainObj);
+                if (anno.online()) {
+                    try {
+                        method.invoke(mainObj, (Object) null);
+                        System.out.println(methodName + " passed: " + anno.desc());
+                    } catch (Exception e) {
+                        System.out.println(methodName + " failed: " +
+                                e.getMessage() + " " + anno.error());
+                    }
+                } else {
+                    System.out.println(methodName + " skipped : offline case");
+                }
+                printAfterMethod(methods, mainObj);
+            }
+        }
+        printAfterClass(methods, mainObj);
+    }
+
+
+    private static void printBeforeMethod(Method[] methods, WebbTests mainObj) {
+        for (Method method1 : methods) {
```

Comment:
mi qich optimal chi, 3 angam for-ov fracel es nuyn methodsneri vra, kareli er mi hat forov fral u arandznacnel before u after test methodnery u before u after class methodnery, u nor en mi for-ov testeri vrayov fral arden,
bayc gaxaparapes chisht a, apres!!